Automattic/wc-calypso-bridge
# Preconfigure product measurement units

### Changes proposed in this Pull Request:



Closes #1165 .
- #1165 






### How to test the changes in this Pull Request:




1. On your WOA dev site, rsync this PR
4. Visit `yoursite.com/_cli`
5. `wp option get woocommerce_dimension_unit` should return `cm`
6. `wp option get woocommerce_weight_unit` should return `kg`
7. Checkout this branch and rsync Calypso Bridge 
8. `wp option get woocommerce_dimension_unit` should return `in`
9. `wp option get woocommerce_weight_unit` should return `oz`
10. Visit `wp-admin/admin.php?page=wc-settings&tab=products` , settings should match



### Other information:




### FOR PR REVIEWER ONLY:
